### ISSUE TITLE
Prisma uses "wrong" encoding / should be able to set charset for connection

### ISSUE THREAD

**Issue by PanMan** [prisma/prisma#12024]
### Bug description

Prisma doesn't always use the right/same character encoding:
Today we migrated our production database, and started getting errors for queries containing emoji:
`Conversion from collation utf8_general_ci into utf8mb4_unicode_ci impossible for parameter", state: "HY000"`
While the database and table use the same character set and encoding, it seems prisma chooses a different charset for the connection to our "new" server.
It would be helpful to be able to specify the charset for the connection in the prisma settings. While I'm sure there is some strange setting in our db server, this would be super helpfull

**Comment by janpio** [comment]
Probably related: https://github.com/prisma/prisma/issues/11842

**Comment by PanMan** [comment]
I agree they are related, and the other issue has more info. I have done quite some digging in the mysql server settings, and can't find the utf8mb3 anywhere, so I'm wondering if it comes from prisma as as (strange) default.

**Comment by haydenbleasel** [comment]
Posted a couple of not super great workarounds over on the other issue: https://github.com/prisma/prisma/issues/11842#issuecomment-2249043507

**Comment by ludralph** [comment]
Closing this in favour of [11842](https://github.com/prisma/prisma/issues/11842)
